src/Modules/ElementUtils/inheritance.js

```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://refract.link/inherited-member/',
+        href: 'http://example.com/inherited-member/',
```

Comment:
I think we should discuss that - would it make sense to point to somewhere in the refract documentation??